site/docs/v1/tech/admin-guide/webauthn.adoc

```diff
@@ -14,7 +14,7 @@ Available since 1.41.0
 
 == Overview
 
-WebAuthn provides the ability for users to authenticate in their browser using the same method they use to unlock their device, like a biometric scan or PIN. WebAuthn has some big security benefits over the traditional username and password. Read on to learn how to configure WebAuthn in your FusionAuth instance. If you'd like more detail on integrating WebAuthn with your FusionAuth instance, check out the link:/docs/v1/tech/guides/webauthn[WebAuthn Developer Guide].
```

Comment:
I saw that @mooreds added a redirect for this old URL after the page moved, but it didn't work on my local site. Is there a reason the old link needs to stay in place?

Reply:
No reason. redirects only work on prod :(  due to how they are implemented.